packages/cw-orch-core/src/env.rs

```diff
@@ -119,7 +119,7 @@ pub fn default_state_folder() -> Result<PathBuf, StdError> {
 impl Default for CwOrchEnvVars {
     fn default() -> Self {
         CwOrchEnvVars {
-            state_file: default_state_folder().unwrap().join("state.json"),
+            state_file: PathBuf::from_str("state.json").unwrap(),
```

Comment:
I'm a bit confused. Does this make the default behaviour that a relative state file is created, i.e. `my_project/state.json` or `~/.cw-orchestrator/state.json`?

Reply:
/path/to/home/.cw-orchestrator/ is created when `state_file` path is relative and do not start with `.` or `..`

Reply:
> `/path/to/home/.cw-orchestrator/` is created when `state_file` path is relative and do not start with `.` or `..`

And it's absolute path without this change

Reply:
So default of state_file
Before:
`state_file: "/path/to/home/.cw-orchestrator/state.json"`
After:
`state_file: "state.json"`

Reply:
amazing!